bundles/org.openhab.binding.spotify/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -349,6 +385,10 @@
 		<item-type>Switch</item-type>
 		<label>Device Shuffle</label>
 		<description>If shuffle is on or off</description>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Power</tag>
```

Comment:
This controls shuffle mod, definitely not `Power`. Could be `Mode` or `MediaControl` (or remove tags at all).